app/components/search.tsx

```diff
@@ -85,8 +86,26 @@ export default function Search({
   const results = tfFinishedLoadingRef.current ? searchResults : baselineSearchResults
   const model = tfFinishedLoadingRef.current ? 'tensorflow' : 'plaintext'
 
+  const hideSearchResults = () => setShowResults(false);
+  const handleBlur = (e) => {
+    // React will recreate the whole search widget if `showResults` changes.
+    // `true -> true` also counts as a change, so make sure that it only
+    // gets mutated when it really should be opened/closed
+    const focusedOnResult = e.relatedTarget?.classList.contains('result-item');
+    if (!focusedOnResult) {
+      hideSearchResults();
+    }
+  }
+  const onQuestionAdded = (title) => {
+      alert('Your question "' + title + '" was added. It might take a while for it to be shown here, as it has yet to be answered.');
```

Comment:
a better text here would be appreciated.
I did it so this function gets passed into the AddQuestion widget because it needs to hide the list of suggested answers after the new one gets added. I wanted to have a single `hideSearchResults()` function to keep things tidy, which would require passing it in anyway, or having some other mechanism in a global function. This seems like the least ugly.

Reply:
'Thanks for asking a new question! "' + title + '" was added to our [suggestion box](https://coda.io/@alignmentdev/ai-safety-info/suggested-questions-66). It might take a while for it to be answered by our writers, but check back in a few months.'